src/Module.h

```diff
@@ -35,6 +35,7 @@ class Module {
   virtual void loop() {}
 
   virtual void connected() {}
+  virtual void enabledChanged(bool tf) {}
```

Comment:
If we have a module disabled before we connect WiFi, connect WiFi and then enable, the module will never get the connected call so might not function properly. Should we trigger a connected call for that module on enable or expect every module to handle this?

Reply:
I think connected() and enabledChanged() are the 'orchestrators' and they can call functions which do the work. 

So function to connect can be called by both and if called by one of them, checks if connected is true and if enabled is true.